docs/root/configuration/http_filters/dynamic_forward_proxy_filter.rst

```diff
@@ -0,0 +1,91 @@
+.. _config_http_filters_dynamic_forward_proxy:
+
+Dynamic forward proxy
+=====================
+
+.. attention::
+
+  HTTP dynamic forward proxy support should be considered alpha and not production ready. Stats
+  as well as circuit breakers are missing and will be added soon.
+
+* HTTP dynamic forward proxy :ref:`architecture overview <arch_overview_http_dynamic_forward_proxy>`
+* :ref:`v2 API reference <envoy_api_msg_config.filter.http.dynamic_forward_proxy.v2alpha.FilterConfig>`
+* This filter should be configured with the name *envoy.filters.http.dynamic_forward_proxy*.
+
+The following is a complete configuration that configures both the
+:ref:`dynamic forward proxy HTTP filter
+<envoy_api_msg_config.filter.http.dynamic_forward_proxy.v2alpha.FilterConfig>`
+as well as the :ref:`dynamic forward proxy cluster
+<envoy_api_msg_config.cluster.dynamic_forward_proxy.v2alpha.ClusterConfig>`. Both filter and cluster
+must be configured together and point to the same DNS cache parameters for Envoy to operate as an
+HTTP dynamic forward proxy.
+
+.. note::
+
+  The HTTP connection manager :ref:`allow_absolute_url
+  <envoy_api_field_core.Http1ProtocolOptions.allow_absolute_url>` parameter has been set to true
+  to allow Envoy to proxy absolute HTTP URLs.
+
+.. attention::
+
+  While configuring an empty :ref:`tls_context <envoy_api_field_Cluster.tls_Context>` on the cluster
+  instructs Envoy to use TLS when connecting to upstream hosts, currently it is not possible to
+  configure per-host TLS configuration parameters including SNI, certificate verification, etc. This
+  will be added in a future change. **This means that the following configuration will not fully
+  validate TLS certificates**. Use with care until full support for per-host validation is
+  implemented.
+
+.. code-block:: yaml
+
+  admin:
+  access_log_path: /tmp/admin_access.log
+  address:
```

Comment:
isn't identation off here? e.g., should be:

```
  admin:
    access_log_path: /tmp/admin_access.log
    address:
...
```

Reply:
Thanks will fix / use a better editor to paste my config